File: Tipos de Dados/Matrizes Associativas/associativas.js
//JavaScript
//Suporte Nativo

// Definição e inicialização
let matriz = {
    "um": 1,
    "dois": 2,
    "tres": 3
};

// Acessar elementos
console.log("Valor para 'dois':", matriz["dois"]);

// Remover um elemento
delete matriz["tres"];

// Iterar pela matriz
for (let chave in matriz) {
    console.log(chave + ": " + matriz[chave]);
}

File: Tipos de Dados/Matrizes Associativas/Associativas.java
//Java
//Uso de Biblioteca
import java.util.HashMap;

public class Associativas {
    public static void main(String[] args) {
        HashMap<String, Integer> matriz = new HashMap<>();

        // Definição e inicialização
        matriz.put("um", 1);
        matriz.put("dois", 2);
        matriz.put("tres", 3);

        // Acessar elementos
        System.out.println("Valor para 'dois': " + matriz.get("dois"));

        // Remover um elemento
        matriz.remove("tres");

        // Iterar pela matriz
        for (String chave : matriz.keySet()) {
            System.out.println(chave + ": " + matriz.get(chave));
        }
    }
}
